src/sentry/seer/anomaly_detection/store_data.py

```diff
@@ -36,7 +36,7 @@
     settings.SEER_ANOMALY_DETECTION_URL,
     timeout=settings.SEER_ANOMALY_DETECTION_TIMEOUT,
 )
-NUM_DAYS = 28
```

Comment:
`NUM_DAYS` isn't used here at all and it's already defined in `utils.py` where it is used. 

`MIN_DAYS` I just moved up from being defined inside a function.